pydriller/metrics/process/new_devs_count.py

```diff
@@ -0,0 +1,87 @@
+"""
+Module that calculates the number of NEW developers that contributed to a file
+"""
+from pathlib import Path
+from pydriller.domain.commit import ModificationType
+from pydriller.repository_mining import RepositoryMining
+from pydriller.metrics.process.process_metric import ProcessMetric
+
+class NewDevsCount(ProcessMetric):
```

Comment:
Why is this not together with the other DevsCount?

Reply:
It was the last metric I have implemented, and done in a different file for simplicity and to test. I will merge them.

Reply:
perfect 👍